Update istio-cni to use logAsJson Flag

erismaster opened the pull request:
The istio-cni chart was missing the logAsJson flag/bool in the chart template. This PR adds that flag, similar to other charts such as istiod.

Signed-off-by: Derrik Campau <dcampau@vmware.com>

**Please provide a description of this PR:**
The helm chart template for istio-cni was not allowing for users to customize if istio-cni should log as JSON output. This PR fixes that :)

erismaster committed:
Update istio-cni to use logAsJson Flag

The istio-cni chart was missing the logAsJson flag/bool in the chart
template. This PR adds that flag, similar to other charts such as
istiod.

Signed-off-by: Derrik Campau <dcampau@vmware.com>

istio-testing commented:
Hi @erismaster. Thanks for your PR.

I'm waiting for a [istio](https://github.com/orgs/istio/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/istio/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=istio%2Fistio).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

erismaster commented:
I am waiting on a company process to finish up for the CLA to pass.

howardjohn commented:
lgtm once we get the CLA covered

erismaster committed:
Update Helm Docs Deprecation Notice

Remove sentence in editing helm docs that said helm was deprecated as
this is no longer accurate.

Signed-off-by: Derrik Campau <dcampau@vmware.com>

howardjohn commented:
/ok-to-test

istio-testing commented:
@erismaster: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:

Test name | Commit | Details | Required | Rerun command
--- | --- | --- | --- | ---
release-notes_istio | 9c06617f0e9c46f54a315418e5d5503fb12822f5 | [link](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/42018/release-notes_istio/1593031975558451200) | true | `/test release-notes`

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>
<!-- test report -->

zirain reviewed (approved):
lgtm, thanks

zirain commented:
/test release-notes

erismaster commented:
@jacob-delgado or @costinm Apologies for a direct ping, but was wondering if this could get the last lgtm to merge :) 
Thanks!
